config/environments/development.rb

```diff
@@ -58,5 +58,16 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.perform_deliveries = true
+  # config.action_mailer.default_url_options = { host: ENV['MAIL_HOST'] }
+  config.action_mailer.smtp_settings = {
+    user_name: ENV['SENDMAIL_USERNAME'],
+    password: ENV['SENDMAIL_PASSWORD'],
+    domain: ENV['MAIL_HOST'],
```

Comment:
Since you are using letter_opener, please remove config.action_mailer.smtp_settings